packages/animation/src/storybookUtils/index.js

```diff
@@ -22,8 +22,8 @@ import { PAGE_HEIGHT, PAGE_WIDTH } from '@web-stories-wp/units';
 /**
  * Internal dependencies
  */
-import Boilerplate from '../../edit-story/output/utils/ampBoilerplate';
-import CustomCSS from '../../edit-story/output/utils/styles';
+import Boilerplate from '../../../../assets/src/edit-story/output/utils/ampBoilerplate';
+import CustomCSS from '../../../../assets/src/edit-story/output/utils/styles';
```

Comment:
These are a blocker. These utils can not simple be copied over as they relay on theme. Have any thoughts @swissspidy ?

Reply:
Fixed!

Reply:
Actually I don't think we need `CustomCSS` here _**at all**_. We could just remove it entirely.

@littlemilkstudio for confirming

Reply:
I think we just use this for documenting the amp portion of the animation parts in storybook, it's not really needed for the animations per se.

So if `CustomCSS` isn't needed for the story preview, then it shouldn't be needed here.

If we really need to, we could just leave these stories & story utils in the root `src/animation` directory or something and just have it be there for storybook documentation/implementations that touch everything else

Reply:
just tested out the storybook implementation for the AMP portions of the animation parts and they seem to work fine without the `CustomCSS` in here at all, so I see no issues with just removing it entirely